src/Management/src/EndpointBase/Metrics/Observer/EventCounterListener.cs

```diff
@@ -121,23 +122,20 @@ private void ExtractAndRecordMetric(string eventSourceName, IDictionary<string,
         long? longValue = null;
         var counterName = string.Empty;
         var labelSet = new List<KeyValuePair<string, object>>();
-        var excludedMetric = false;
+
         string counterDisplayUnit = null, counterDisplayName = null;
         foreach (var payload in eventPayload)
         {
-            if (excludedMetric)
-            {
-                break;
-            }
-
             var key = payload.Key;
             switch (key)
             {
                 case var kn when key.Equals("Name", StringComparison.OrdinalIgnoreCase):
                     counterName = payload.Value.ToString();
-                    if (_options.ExcludedMetrics.Contains(counterName))
+                    if ((_options.IncludedMetrics.Any()
+                        && !_options.IncludedMetrics.Contains(counterName))
+                        || _options.ExcludedMetrics.Contains(counterName))
```

Comment:
When an entry exists in both include/exclude lists, it will be included. Is that intentional, or should the exclude list win over inclusions? Either way, it would be helpful to document which list has precedence.

Reply:
Yes, Include takes precedence in mind - since its more exclusive.

Reply:
https://github.com/SteeltoeOSS/Documentation/issues/288

Reply:
I didn't mean to create external documentation. Just a simple line like "In case of conflict, see [ref IncludedMetrics] takes precedence." on the `ExcludedMetrics` property should be sufficient.

Reply:
Seems fair to put it both places. This new property definitely does need to be in the docs regardless of how it works

Reply:
self documenting code